server/mqtt.go

```diff
@@ -1142,6 +1142,9 @@ func (s *Server) mqttCreateAccountSessionManager(acc *Account, quitCh chan struc
 			quitCh: quitCh,
 		},
 	}
+	if !opts.MQTT.disableRetainedMessageCache {
```

Comment:
If we don't want to add an option (as to not expose in option parsing), what you could do is in this file, declare this boolean:
```
var testDisableRMSCache bool
```
And here, you would do:
```
	if !testDisableRMSCache {
		as.rmsCache = &sync.Map{}
	}
```
Now see test to know how to activate.

Reply:
Oh silly me, I forgot that even the cluster runs all in the same process! Of course, thank you. Raciness is not an issue since this is only used in benchmarks (no parallel execution)